app/models/repository_table.rb

```diff
@@ -0,0 +1,60 @@
+class RepositoryTable < ActiveRecord::Base
```

Comment:
Should we rename this to `RepositoryTableState`? I mean, just to point out specifically that this is a table just to store state data.